core/src/utilities/helpers/iframe-helpers.js

```diff
@@ -216,6 +216,13 @@ class IframeHelpersClass {
     const iframe = document.createElement('iframe');
     iframe.src = ViewUrlDecorator.hasDecorators() ? ViewUrlDecorator.applyDecorators(viewUrl) : viewUrl;
     if (allowRules) {
+      allowRules.forEach((rule, index) => {
+        if (rule.indexOf(';') != -1) {
+          allowRules[index] = rule;
+        } else {
+          allowRules[index] = rule + ';';
+        }
```

Comment:
Works well 👍 
Have a suggestion here, which could make this a bit easier and save some lines:
```suggestion
        allowRules[index] = rule + (rule.includes(';') ? '' : ';');
```